.travis.yml

```diff
@@ -1,48 +1,38 @@
 language: python
-
-
 matrix:
   include:
     - env: TOXENV=docs
     - env: TOXENV=pep8
     - env: TOXENV=py3pep8
+    - env: TOXENV=isort
```

Comment:
A new Travis CI job has been added that checks that all the imports in the project are in a sorted order.

The configuration for this is stored in `setup.cfg` in the `isort` section.